spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SessionsController, type: :controller do
+  describe 'GET#new' do
+    context 'when user_id session not exists' do
```

Comment:
how about changing the context a bit @gutakk, we usually use the `give something term`

```
context 'given the user_id session' do


context 'given NO user_id session' do
```

Reply:
Good point. I not sure if we can use `when`?
ref : https://compass.nimblehq.co/development/code-conventions/rspec/#formatting

Reply:
Fixed in d85aa0494bcbbe6eb1c1b7de8eaa64b6c4c6b159